application/jaxrs-hazelcast-quarkus/src/test/java/webapp/tier/service/HazelcastServiceStatusTest.java

```diff
@@ -0,0 +1,64 @@
+package webapp.tier.service;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+class HazelcastServiceStatusTest {
+
+	@Inject
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5997249)